packages/core/src/dialects/postgres/query-generator.js

```diff
@@ -553,8 +553,9 @@ export class PostgresQueryGenerator extends PostgresQueryGeneratorTypeScript {
     return `ALTER FUNCTION ${oldFunctionName}(${paramList}) RENAME TO ${newFunctionName};`;
   }
 
+  // TODO [>7]: remove usage of removeTicks in favor of something that does not mishandle single quotes
   pgEscapeAndQuote(val) {
-    return this.quoteIdentifier(removeTicks(this.escape(val), '\''));
+    return this.quoteIdentifier(removeTicks(this.escape(val), `'`));
```

Comment:
based on where this is used, can't the usages of this function just be replaced by quoteIdentifier?

Reply:
I'll check, but I believe I didn't do anything to this because there was a PR in progress with `pgEnumDrop` that could remove this function. Not sure which one though, or if I remember this correctly